zebra-consensus/src/primitives/halo2/tests/vectors/orchard_shielded_data.rs

```diff
@@ -0,0 +1,28 @@
+//! Orchard shielded data (with Actions) test vectors
```

Comment:
Efficiency: Can you please put large test vectors in `zebra-test`, so they are only downloaded by test builds?

Reply:
Done!

2914acfa